CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+- **DON'T** submit PRs that alter licensing related files or headers. If you believe there's a problem with them, file an issue and we'll be happy to discuss it.
+
+### Breaking Changes
+
+Contributions must maintain API signature and behavioral compatibility. Contributions that include breaking changes will be rejected for the current release and can only be considered for the next major release. Please file an issue to discuss your idea or change if you believe that it may affect compatibility.
+
+### Suggested Workflow
+
+We use and recommend the following workflow:
+
+1. Create an issue for your work.
+   - You can skip this step for trivial changes.
+   - Reuse an existing issue on the topic, if there is one.
+   - Get agreement from the team and the community that your proposed change is a good one.
+   - Clearly state that you are going to take on implementing it, if that's the case.
+2. Create a personal fork of the repository on GitHub (if you don't already have one).
+3. In your fork, create a branch off of main (if your change affects the current version) or the release-branch you want to contribute to (if you want to contribute to a future release).
+   - Name the branch according to our [branching guide](#branching-guide).
+   - Branches are useful since they isolate your changes from incoming changes from upstream. They also enable you to create multiple PRs from the same fork.
+4. Make and commit your changes to your branch.
+   - You can run static code checks by running `npm run lint`.
+   - You can run tests locally by running `npm run test:local:node`.
+   - Please follow our [Commit Messages](#commit-messages) guidance.
+5. Add new tests corresponding to your change, if applicable.
+6. Build the repository with your changes.
+   - Make sure that the builds are clean.
+   - Make sure that the tests are all passing, including your new tests.
+7. Make sure your branch has all changes from the main/release branch of the source repository.
+8. Increment the npm package version in the package.json (adhere to [semantic versioning](https://semver.org/)). Run `npm i` to make sure the change is propagated to package-lock.json.
+9. Create a pull request (PR) against the repository's **main** branch or the **release/\*** branch of the release you want to contribute to.
+   - State in the description what issue or improvement your change is addressing.
+   - Check if all the continuous integration checks are passing.
+10. Wait for feedback or approval of your changes from the team.
+11. When the PR is approved and all checks are green, your PR will be merged by the team.
+    - This will automatically trigger our continuous deployment pipeline, which builds and publishes a new version.
+    - You can delete the branch you used for making the change.
+
+### Branching Guide
+
+Branches must match the following pattern:
+
+```text
+<prefix>/[<issue-id>]<description>
+```
+
+- `<prefix>`: must be one of `chore`, `feature` or `bugfix`
```

Comment:
prefix ist in <> und die anderen nicht

Reply:
Ähm, ja, weil prefix der Placeholder ist, und die anderen potentielle Werte.

Reply:
ich versteh den Untereschied nicht ..